tests/gopherjsvendored_test.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/sh
+# Don't run this file directly. It's executed as part of TestGopherJSCanBeVendored.
+
+set -e
+
+tmp=$(mktemp -d "${TMPDIR:-/tmp}/gopherjsvendored_test.XXXXXXXXXX")
+
+cleanup() {
+    rm -rf "$tmp"
+    exit
+}
+
+trap cleanup EXIT HUP INT TERM
+
+# copyGoPackage copies Go package with import path $1 to directory $2.
+# The target directory is created if it doesn't exist.
+copyGoPackage() {
+    mkdir -p "$2"
+    pkgDir=$(go list -f '{{.Dir}}' "$1")
+    # Copy all files (not directories), other than ones that start with "." or "_".
+    for f in $(find -H "$pkgDir" -maxdepth 1 -name "[^._]*" -type f); do
+        cp "$f" "$2"
+    done
+}
+
+# Make a hello project that will vendor GopherJS.
+mkdir -p "$tmp/src/example.org/hello"
+echo 'package main
```

Comment:
Just a minor thought, but the heredoc syntax often reads a bit cleaner in these multi-line situations because the redirect is on the same line:

```bash
cat <<EOD > "$tmp/src/example.org/hello/main.go"
package main

import "github.com/gopherjs/gopherjs/js"

func main() {
    js.Global.Get("console").Call("log", "hello using js pkg")
}
EOD 
```

Reply:
Thanks for the suggestion. I wasn't familiar with this syntax, but I'll consider it next time.